test/specialarrays.jl

```diff
@@ -52,6 +55,33 @@ using StaticArrays
             @test Xt_invA_X(A, Y) isa SMatrix{10, 10, Float64}
             @test Xt_invA_X(A, Y) ≈ Matrix(Y)' * (Matrix(A) \ Matrix(Y))
         end
+
+        # TODO: Fix for `PDMat` and `PDiagMat`
```

Comment:
is there an issue tracking this TODO?

Reply:
No, no issue. But it will be fixed by https://github.com/JuliaStats/PDMats.jl/pull/183 🙂